neoload/commands/test_results.py

```diff
@@ -41,7 +41,8 @@ def load_from_file(file):
 @click.option('--external-url', 'external_url', help="URL to an external system, for example the CI job's link")
 @click.option('--external-url-label', 'external_url_label',
               help="Label to describe the external URL, for example the CI name or job ID")
-def cli(command, name, rename, description, quality_status, junit_file, file, filter, external_url, external_url_label):
+@click.option('--lock/--unlock', default=False, help="Protect a specific result to avoid automatic or accidental manual deletion")
```

Comment:
I think that default value should be None !!!! Because as-is, if you execute the command "neoload test-results --description JeVeuxJusteUpdateLaDescription patch 4c17562e-b7e1-417e-96ba-5b564d07a53b", then the result will be unlocked !!

Reply:
Yes you are right, i'll fix that immediately ! Thx for your review :)